girder/utility/gridfs_assetstore_adapter.py

```diff
@@ -65,6 +65,7 @@ def validateInfo(doc):
                 ('uuid', pymongo.ASCENDING),
                 ('n', pymongo.ASCENDING)
             ], unique=True)
+            chunkColl.create_index([('n', pymongo.ASCENDING)])
```

Comment:
In using Mongo version 3.2.9, I was getting an error when trying to download a large file that the data could not fit in memory and that an index was needed.  The lines
```
 cursor = self.chunkColl.find({
            'uuid': file['chunkUuid'],
            'n': {'$gte': n}
        }, projection=['data']).sort('n', pymongo.ASCENDING)
```
were not using the compound index.  Adding this index solves that.